distribution/kernel/carbon-home/repository/resources/conf/templates/repository/conf/carbon.xml.j2

```diff
@@ -517,6 +517,7 @@
 	<!--Option to disable storing of tokens issued by STS-->
 	<DisableTokenStore>false</DisableTokenStore>
 
+    <STSCallBackHandlerName>{{sts_call_back_handler.name | default('org.wso2.carbon.identity.provider.AttributeCallbackHandler')}}</STSCallBackHandlerName>
```

Comment:
Change the key "sts_call_back_handler.name" to "sts.callback_handler". 
It is better to add the default value to default.json.